plugins/vue-inter.js

```diff
@@ -56,6 +57,7 @@ const messages = {
       signOut: 'Se déconnecter',
       signIn: 'Se connecter',
       slogan: 'Une application décentralisée de notes',
+      downloadNotes: 'Notes de téléchargement',
```

Comment:
```suggestion
      downloadNotes: 'Télécharger les notes',
```

It's a better translation into French